FlowCrypt/src/test/java/com/flowcrypt/email/api/retrofit/response/model/ClientConfigurationTest.kt

```diff
@@ -18,11 +18,13 @@ class ClientConfigurationTest {
     val clientConfiguration = ClientConfiguration(
       disallowPasswordMessagesForTerms = listOf(
         "droid",
+        "test-feature",
         "[Classification: Data Control: Internal Data Control]",
       )
     )
 
     val regex = requireNotNull(clientConfiguration.getDisallowPasswordMessagesForTermsRegex())
+    println(regex)
```

Comment:
this line probably was needed only for debug and can be removed?

Reply:
Yes, I will remove it via a separate PR.